src/main/resources/crafter/studio/studio-config.yaml

```diff
@@ -108,6 +108,10 @@ studio.configuration.site.dataSources.configBasePath: /config/studio/data-source
 studio.configuration.site.dataSources.configFileName: cmis-config.xml
 # Destroy site context url for preview engine
 studio.configuration.site.preview.destroy.context.url: http://localhost:8080/api/1/site/context/destroy.json?crafterSite={siteName}
+# Location pattern where default dependency resolver specific configuration for is stored
+studio.configuration.default.dependencyResolver.configBasePath: /configuration/dependency
```

Comment:
Shouldn't it be better to specify these properties as one, since we're only using the path?

Reply:
All configuration files have properties like that (filename and path in separate properties)